### PR TITLE
GUI_CreateBalancePage

### PR DESCRIPTION
**Pull request by dTarasova**

### CODE REVIEW

File: Clients/GUI/Pages/Balance.razor [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+    protected override async Task OnInitializedAsync()
+    {
+        errorOccured = false;
+        isPageLoading = true;
+
+        try
+        {
+            userBalance = new UserBalance() { UserId = await Storage.GetItemAsync<Guid>("id") };
+            userBalance = await BrokerManager.GetUserBalance(userBalance.UserId);
```
Comment (ifelseelif) [inline Clients/GUI/Pages/Balance.razor]: what's going on here?

File: Clients/GUI/Pages/InstrumentInfo.razor [hunk 2]
```diff
@@ -20,7 +20,13 @@
             <ChartJsLineChart @ref="lineChartJs" Config="@lineConfig" Width="300" Height="150" />
         </div>
     </div>
+    
     <div class="col-md-6">
+        <div class="form-row input-group-prepend">
+            <label class="col-9 text-right">Current balance:  @currentBalance  @currency</label>
+            <input type="button" class="btn-link col-md-3" value="Increase balance" @onclick="GoToBalancePage">
```
Comment (nmamro) [inline Clients/GUI/Pages/InstrumentInfo.razor]: Make this also btn-primary I guess, so it will look better

**Review by nmamro — APPROVED** [review]

File: Clients/GUI/Pages/InstrumentInfo.razor (hunk 2 above)
Comment (dTarasova) [reply Clients/GUI/Pages/InstrumentInfo.razor]: made btn-light. Primary attracts too much attention

File: Clients/GUI/Pages/Balance.razor (hunk 1 above)
Comment (dTarasova) [reply Clients/GUI/Pages/Balance.razor]: Something strange, thanks :) fixed

**Review by ifelseelif — APPROVED** [review]

**Pull request merged**